Reflections/Week_4_Reflection.md

```diff
@@ -0,0 +1,19 @@
+# Weekly Reflection- Week Four
+
+1. What have you accomplished this week? (Please list each accomplishment, and explain)
+  * I have worked more on expanding an interior. I have made the interior accessible in game. I have moved several quest-important items and an NPC to the interior.
+
+![Interior](WeeklyScreenshots/WeekFourInteriorOne.png)
+![Interior](WeeklyScreenshots/WeekFourInteriorTwo.png)
+![Interior](WeeklyScreenshots/WeekFourInteriorThree.png)
+![Interior](WeeklyScreenshots/WeekFourInteriorFour.png)
+
+2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
+  * I have had to study for midterms and make time to complete them. I have two papers I am working on simultaneously.
+
+3. What do you plan to accomplish in the following week? (Please list at least 3 concrete "S.M.A.R.T." goals, along with estimated number of hours to complete, or day to complete)
+  * I have several goals from last week that I still need to work towards, like the dialogue between Mario and his boss. That, in reference to how NPC interaction works in accordance to dialogue, should be easy. It should take around two hours. I will create perks for Mario. This should take an hour. I will finish up the main interior of the mod and work on a secondary one. This could take upwards of five hours.
```

Comment:
Life happens! I think these are great smart goals. Keep chippin' away at it, you got this!